types/react-relay/index.d.ts

```diff
@@ -131,15 +132,14 @@ export interface PageInfo {
 }
 export interface ConnectionData {
     edges?: ReadonlyArray<any>;
-    pageInfo?: Partial<PageInfo>;
+    pageInfo?: Partial<PageInfo> | null;
 }
 export type RelayPaginationProp = RelayProp & {
     hasMore(): boolean;
     isLoading(): boolean;
     loadMore(
         pageSize: number,
-        callback: (error?: Error) => void,
-        options?: RefetchOptions
+        callback?: (error?: Error) => void
```

Comment:
The docs didn't reflect reality, so I've sent a PR to update the docs https://github.com/facebook/relay/pull/2619.